com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -763,13 +763,14 @@ private static File createUniversalApks(Project project, File aab, File outDir,
         String keystoreAlias = getKeystoreAlias(project);
         String keyPasswordFile = getKeyPasswordFile(project);
 
-        try {
-            File bundletool = new File(Bob.getLibExecPath("bundletool-all.jar"));
+        File bundletool = new File(Bob.getLibExecPath("bundletool-all.jar"));
```

Comment:
I decided to upgrade from our very old bundletool version to the latest one: https://github.com/google/bundletool/releases/tag/1.14.0